packages/voter-dapp/src/ActiveRequests.js

```diff
@@ -140,7 +141,7 @@ function ActiveRequests({ votingAccount, votingGateway, snapshotContract }) {
         setPendingRequests(nonBlacklistedRequests);
       }
     }
-  }, [allPendingRequests, IDENTIFIER_BLACKLIST, showSpamRequests]);
+  }, [allPendingRequests, hexToUtf8, showSpamRequests]);
```

Comment:
modified this due to build warnings. most warnings are gone now

Reply:
Super strange that this `useEffect()` hook would depend on the value of `hexToUtf8`, which is a function?

Reply:
its not that it doesnt depend on it, but react warns that mutations to `IDENTIFIER_BLACKLIST` will not re-render component, so it isnt needed
![image](https://user-images.githubusercontent.com/4429761/109739315-6cc41500-7b97-11eb-8c48-3b9cea1f2805.png)